config/config_test.go

```diff
@@ -107,7 +107,7 @@ func TestGetSamplerTypes(t *testing.T) {
 		Sampler = "DeterministicSampler"
 		SampleRate = 2
 
-	[SamplerConfig.dataset1]
+	[SamplerConfig.'dataset 1']
```

Comment:
might be worth adding a comment to the `rules.toml` to make sure people know about escaping the dataset name if it contains a space